project/plugin.sbt

```
@@ -3,6 +3,7 @@ addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.4.10")
 addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "1.2.0")
 addSbtPlugin("org.portable-scala" % "sbt-scala-native-crossproject" % "1.2.0")
 addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.4.4")
+addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.5.0")
 val sbtTypelevelVersion = "0.4.18"
 addSbtPlugin("org.typelevel" % "sbt-typelevel-ci-release" % sbtTypelevelVersion)
 addSbtPlugin("org.typelevel" % "sbt-typelevel-settings" % sbtTypelevelVersion)
```

Now that we have a scalafmt config, if we're willing to throw in copyright headers than we can upgrade to `sbt-typelevel` proper:
```suggestion
addSbtPlugin("org.typelevel" % "sbt-typelevel" % "0.4.18")
```